### ISSUE TITLE
Cache error

### ISSUE THREAD

**Issue by KurtzPT** [ZeroQI/Hama.bundle#218]
With the latest version on main, while adding new episodes on One Piece:

`2018-06-19 11:27:12,051 (f9c) :  CRITICAL (core:574) - Error trimming cache (most recent call last):
  File "C:\Program Files (x86)\Plex\Plex Media Server\Resources\Plug-ins-fd05be322\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\components\caching.py", line 264, in trim
    del self._info[key]
KeyError: u'http://thetvdb.plexapp.com/banners/seasons/79481-1-8.jpg'`

**Comment by ZeroQI** [comment]
Look like a Plex error, not in my code, and the url is valid and show a Death note poster...
Please attach Death note serie logs located in agent data folders

**Comment by KurtzPT** [comment]
I did delete all files in DataItems folder before starting a scan, is it because of that? 

I only have the ".scanner.log" and the ".filelist.log" logs for Death Note, there's no agent logs.

**Comment by ZeroQI** [comment]
Yeah that would explain as you deleted internal plex files at the same time...
if you refresh metadata or match the series again it will create them again.
